### DISCUSSION TITLE
~/.config/i3/config

### DISCUSSION THREAD

**Discussion by WgpArch** [i3/i3#5459] [Q&A] [answered]
I deleted all content of ~/.config/i3/config and uninstalled all i3 packages,then I reinstalled it again but ran into problems cause the  ~/.config/i3/config is empty. I ran i3-config wizard as root but it did restore anything in the config file,I would like to where I can get the original default config so I can put it back in ~/.config/i3/config I found other peoples configs on Github but they are theirs and not the original default.I know I messed up big time please help me.

**Comment by orestisfl** [comment, marked as answer]
To start with, don't run `i3-config-wizard` as root. `sudo` is not to be used unless your user doesn't have permissions for what you are trying to do. Specifically in this case, running the wizard as root would place the config files in root's home directory.

So, to begin with, run the wizard as your normal user and see if it does what you want it to do.

**Reply by WgpArch** [reply]
When reinstall i3-wm I run i3-cofig-wizard and it tells it already exist I go to /etc/ and try run it with execute nothing happens I to to open it with geany then it shows my openbox rc.xml,I try to open it with libreoffice writer to get to content but it tells me the file is corrupted. I fount to folders in /etc/ with content in and copied it to ~/.config/i3/config but when I restart and login to i3 the keybindings does not work only mod1+space+return opens my lxterminal.

**Reply by orestisfl** [reply]
What's the output of `ls -lah ~/.i3; ls -lh ~/.config/i3`?

**Reply by WgpArch** [reply]
[wgparch@archlinux ~]$ ls -lah ~/.i3; ls -lh ~/.config/i3
ls: cannot access '/home/wgparch/.i3': No such file or directory
total 8.0K
-rw-r--r-- 1 wgparch wgparch 7.7K Mar 26 22:26 config
[wgparch@archlinux ~]$ 
It will show this cause I again uninstalled it

**Reply by WgpArch** [reply]
My /etc/skel folder is also empty

**Reply by orestisfl** [reply]
Then you just need to `rm -rf ~/.config/i3` and the wizard should show up next time you start i3

**Reply by WgpArch** [reply]
Ok thanks I will restart now and look.

**Reply by orestisfl** [reply]
If not, paste the output of `cat /etc/i3/config`

**Reply by WgpArch** [reply]
Ok thanks I will restart now and look.

**Reply by WgpArch** [reply]
Thank you thank you it worked wow thanks
